zabbix_metrics_watcher.py

```diff
@@ -214,14 +216,32 @@ def run(zabbix_host, osbs_master, config, instance):
 
             logger.info("Found build %s in %s", build_name, status)
             build = Build(build_name, cmd_base)
-            if status == 'Pending':
+            if status == 'New':
+                if list(builds_in_new) == []:
+                    builds_in_new_time = datetime.datetime.now()
+                else:
+                    timediff = int((datetime.datetime.now() - builds_in_new_time).total_seconds())
+                    logger.info("%s build(s) are in New for %s sec", len(builds_in_new), timediff)
+                    _send_zabbix_message(zabbix_host, osbs_master, "new_duration", timediff)
```

Comment:
Well, couldn't you store the same information you were already going to store, just based on the build that has spent the longest in New?

The point is to make sure that some particular build has spent the time we're reporting in New, not that there have been builds in New, perhaps transiently, for some amount of time.